scalafix-tests/src/test/resources/checkSyntax/RemoveXmlLiterals.source

```diff
@@ -130,3 +130,21 @@ import scala.xml.quote._
 object A {
   xml"<a>${1}${2}</a>"
 }
+
+<<< patterns: not supported
```

Comment:
Comment them out. I couldn't find a way to do skipped tests like before, but maybe we can add it later in the future.